web/templates/host.html.tmpl

```diff
@@ -7,15 +7,16 @@
             <dd class="inline">{{ .Host.Name }}</dd>
             {{ $Env := index .Host.TrentoMeta "trento-sap-environment" }}
             {{ $Land := index .Host.TrentoMeta "trento-sap-landscape" }}
-            {{ $SAPSys := index .Host.TrentoMeta "trento-sap-system" }}
             <dt class="inline">Environment</dt>
             <dd class="inline"><a href="/environments/{{ $Env }}">{{ $Env }}</a></dd>
             <dt class="inline">Landscape</dt>
             <dd class="inline"><a href="/landscapes/{{ $Land }}?environment={{ $Env }}">{{ $Land }}</a></dd>
             <dt class="inline">SAP System</dt>
-            <dd class="inline"><a
-                        href="/sapsystems/{{ $SAPSys }}?environment={{ $Env }}&landscape={{ $Land }}">{{ $SAPSys }}</a>
+            <dd class="inline">
+                {{- range $i, $v := split (.Host).GetSIDsString ","}}{{- if $i }},{{- end }} <a
```

Comment:
Do we need this `.Host.GetSIDsSting` function?
It will return the same value we have in the metadata, right?
I mean, why simply not: `split (index .Host.TrentoMeta "trento-sap-systems) ","`
And we get rid off `GetSIDsString` function from the `Host` struct